docs/guides/installing-on-linux/setting-up-ssl.md

```diff
@@ -7,43 +7,64 @@ sidebar_position: 2
 
 # Setting up SSL for Ant Media Server
 
-Setting up SSL is a mandatory task when requesting access to the microphone and camera. Also, you need to enable HTTPS and WSS (WebSocket Secure) for Chrome to run WebRTC and WebSocket applications.
+Setting up SSL is a mandatory task when requesting access to the microphone and camera. Also, you need to enable HTTPS and WSS (WebSocket Secure) for Chrome to run WebRTC and WebSocket applications. In addition, developers want to serve their content with a secure connection as well.
+There are several options to get the SSL certificate. Please choose the one appropriate for you.
 
-In addition, developers want to serve their content with a secure connection as well. The script in this document installs **Let's Encrypt** SSL certificate, however you can also use your own SSL installation method.
+## Option 1 Enabling SSL from the Web Panel
+In previous versions, configuring SSL involved intricate steps, such as accessing the server through SSH and executing the `enable_ssl.sh` script from the installation directory `usr/local/antmedia`.
 
-First, create an `A` record for your domain name in your DNS records. This way, your domain name will be resolved to your server's public IP address. Note that this guide is for Ubuntu systems, but there are several guides on the internet for other Linux distributions as well.
+However, with the release of Ant Media Server version 2.6.2, we have streamlined the SSL enablement process, allowing users to seamlessly secure their media server directly from the AMS Web Panel. This empowers users to enable SSL with utmost ease and convenience.
+
+- After [installaling the Ant Media Server](https://antmedia.io/docs/guides/installing-on-linux/installing-ams-on-linux/), login to the web panel and navigate to SETTINGS > SSL.
+![](@site/static/img/ssl-webpanel/ssl-settings.png)
+
+- In the drop-down select box named Type, choose among the various options to enable SSL like [using your own domain](https://antmedia.io/docs/guides/installing-on-linux/setting-up-ssl/#option-2-create-lets-encrypt-certificate-with-http-01-challenge), [free subdomain of antmedia.cloud](https://antmedia.io/docs/guides/installing-on-linux/setting-up-ssl/#option-1-gets-a-free-subdomain-and-install-ssl-with-lets-encrypt), or to [import your own certificate](https://antmedia.io/docs/guides/installing-on-linux/setting-up-ssl/#option-3-import-your-custom-certificate) and then click Activate to enable the SSL and restart your server.
+![](@site/static/img/ssl-webpanel/ssl-options.png)
+
+- This will start to enable SSL for your Ant Media Server.
+![](@site/static/img/ssl-webpanel/enabling-ssl.png)
+
+- The Ant Media Server instance will restart and the server can now be accessed securely with SSL enabled.
+![](@site/static/img/ssl-webpanel/ssl-status.png)
+
```

Comment:
Maybe we can have "Option 2: Install SSL using the Terminal" and then sub-options for free subdomain, own domain, import certificate..etc..